# Insights de questionário escolar

Exploração de dados de informações de um questionário aplicado à alunos de primeiro ano (simulação).

## Organização dos dados

In [100]:
import pandas as pd
questionario = pd.read_csv("../data/questionario_estudantil.csv", index_col=0)
print(questionario.head())

   Turma Genero  Idade  Altura  Peso  Filhos Fuma Tolerancia  Exercicios  \
Id                                                                         
1      A      F     17    1.60  60.5       2  NAO          P           0   
2      A      F     18    1.69  55.0       1  NAO          M           0   
3      A      M     18    1.85  72.8       2  NAO          P           5   
4      A      M     25    1.85  80.9       2  NAO          P           5   
5      A      F     19    1.58  55.0       1  NAO          M           2   

    Cinema OpCine  TV OpTV  
Id                          
1        1      B  16    R  
2        1      B   7    R  
3        2      M  15    R  
4        2      B  20    R  
5        2      B   5    R  


## Tabela de frequência para variável Gênero

In [101]:
frequencia_genero = questionario.groupby("Genero")["Genero"].value_counts()

# new dataframe with the frequency of each gender
new_df = frequencia_genero.to_frame(name="Frequencia")
new_df["Acumulado"] = new_df["Frequencia"] / new_df["Frequencia"].sum()

# pivot table with the frequency of each gender
pivot_table = new_df.pivot_table(index="Genero", values=["Frequencia", "Acumulado"], aggfunc = sum, margins=True, margins_name="Total")
print(pivot_table[["Frequencia", "Acumulado"]])

        Frequencia  Acumulado
Genero                       
F               37       0.74
M               13       0.26
Total           50       1.00


### Resultado
Observa-se que 74% dos alunos são do gênero feminino.